CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.4.1] - 2019-10-08
```

Comment:
This should be a minor release. It's not a bugfix.

Reply:
I agree. Sadly I've noticed your comment only after merge. Will follow your recommendation in the future. Thanks!